cli/azd/CHANGELOG.md

```diff
@@ -1,9 +1,13 @@
 # Release History
 
-## 0.1.0-beta.6 (Unreleased)
+## 0.2.0-beta.1 (2022-09-14)
 
-### Breaking Changes
+### Features Added
 
+- [[#226]](https://github.com/Azure/azure-dev/issues/226) Added support for Terraform for infrastructure as code (IaC), plus a new Terraform template for Python workloads.
```

Comment:
Might be worth it to split adding terraform and adding the terraform template (https://github.com/Azure/azure-dev/pull/646)